<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseSamba</class>
 <widget class="QWidget" name="ChooseSamba">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="printerL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">printer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string comment="@title:group">Windows Printer via SAMBA</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string notr="true">smb://</string>
          </property>
          <property name="buddy">
           <cstring>addressLE</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="KLineEdit" name="addressLE"/>
          </item>
          <item>
           <widget class="QPushButton" name="browsePB">
            <property name="text">
             <string comment="@action:button">Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string notr="true">smb://</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string comment="@label samba URL markup example, this is preceeded by a label containing smb://">[workgroup/]server[:port]/printer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string comment="@title:group">Authentication</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="@label:textbox">Username:</string>
          </property>
          <property name="buddy">
           <cstring>usernameLE</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="@label:textbox">Password</string>
          </property>
          <property name="buddy">
           <cstring>passwordLE</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KLineEdit" name="usernameLE">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="KLineEdit" name="passwordLE">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string comment="@option:radio">Prompt user if password is required</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QRadioButton" name="authenticationCB">
          <property name="text">
           <string comment="@option:radio">Set authentication details now</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="verifyPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="@action:button">Verify</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>addressLE</tabstop>
  <tabstop>browsePB</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>authenticationCB</tabstop>
  <tabstop>usernameLE</tabstop>
  <tabstop>passwordLE</tabstop>
  <tabstop>verifyPB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>authenticationCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authenticationCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>usernameLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authenticationCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>authenticationCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
